autopilot/betweenness_centrality.go

```diff
@@ -50,12 +54,20 @@ func (q *queue) empty() bool {
 // shortest paths starting or ending at that node. This is a useful metric
 // to measure control of individual nodes over the whole network.
 type BetweennessCentrality struct {
+	workers    int
 	centrality map[NodeID]float64
 }
 
 // NewBetweennessCentralityMetric creates a new BetweennessCentrality instance.
-func NewBetweennessCentralityMetric() *BetweennessCentrality {
-	return &BetweennessCentrality{}
+// Users can specify the number of workers to use for calculating centrality.
+func NewBetweennessCentralityMetric(workers int) *BetweennessCentrality {
+	// There should be at least one worker.
+	if workers < 1 {
```

Comment:
return error when invalid value is passed? for configuration imo it's better to fail than silently succeed